app/javascript/react/components/UserShowContainer.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react"
+import UserShowTile from "./UserShowTile"
+
+const UserShowContainer = (props) => {
+  const [user, setUser] = useState({})
+
+  const getUser = async () => {
+    try {
+      const userId = props.match.params.userId
+      const response = await fetch(`/api/v1/users/${userId}`)
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const fetchedUser = await response.json()
+      setUser(fetchedUser)
+    } catch(err) {
+      console.error(`Error in fetch: ${err.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  return (
+    <UserShowTile
+      id={user.id}
+      name={user.username}
+      zip={user.zip}
+    />
```

Comment:
lets not have a UserShowTile. I know you've done this in the past, but I think its an uneccessary separation of concerns. Just have your `UserShowTile` JSX right in here